src/tests/ftest/util/ior_test_base.py

```diff
@@ -93,12 +93,14 @@ def get_job_manager_command(self):
 
         """
         # Initialize MpioUtils if IOR is running in MPIIO mode
-        if self.ior_cmd.api.value == "MPIIO":
+        if self.ior_cmd.api.value in ["MPIIO", "DAOS"]:
```

Comment:
I know its minor, but can we also update the comment (line 95) to match the new code?